config/initializers/link_checker_api.rb

```diff
@@ -0,0 +1,6 @@
+require "gds_api/link_checker_api"
+
+Whitehall.link_checker_api_client = GdsApi::LinkCheckerApi.new(
+  Plek.find("link-checker-api"),
+  bearer_token: ENV["LINK_CHECKER_API_BEARER_TOKEN"] || "example"
```

Comment:
We don't need a bearer token when interacting with the Link Checker API since it's internal.